docker/prod-server/etebase-server.ini

```diff
@@ -3,6 +3,9 @@ secret_file = secret.txt
 debug = false
 static_root = ./data/static
 media_root = ./data/media
+static_url = /static/
+media_url = /user-media/
+redis_uri = redis://localhost:6379
```

Comment:
@vitaly512 in prod, redis url should be inserted through an environment variable